frontend/src/app/features/work-packages/routing/partitioned-query-space-page/partitioned-query-space-page.component.ts

```diff
@@ -149,24 +146,20 @@ export class PartitionedQuerySpacePageComponent extends WorkPackagesViewBase imp
     // Load query on URL transitions
     this.queryParamListener
       .observe$
-      .pipe(
-        this.untilDestroyed(),
-      ).subscribe(() => {
-      /** Ensure we reload the query from the changed props */
+      .pipe(this.untilDestroyed())
+      .subscribe(() => {
+        /** Ensure we reload the query from the changed props */
         this.currentQuery = undefined;
-        this.refresh(true, true);
+        void this.refresh(true, true);
       });
 
-    // Update title on entering this state
-    this.unRegisterTitleListener = this.$transitions.onSuccess({}, () => {
-      this.updateTitle(this.querySpace.query.value);
-    });
-
-    this.querySpace.query.values$().pipe(
-      this.untilDestroyed(),
-    ).subscribe((query) => {
-      this.onQueryUpdated(query);
-    });
+    this.querySpace.query.values$()
+      .pipe(this.untilDestroyed())
+      .subscribe((query) => {
+        // Update the title whenever the query changes
+        this.updateTitle(query);
+        this.currentQuery = query;
+      });
```

Comment:
I found multiple callbacks which all kind of want to update something whenever a query changes. The one, I deleted (prev. line 161), was conflicting badly, displaying a fallback title for approx. 0.5s. 
If anyone sees something I didn't (like a reason we needed multiple callbacks). please alert me. ^^

Reply:
I think the first subscription ensured that the title is "set back" to the query title when you move back from the split view. But that was a hack in the first place. Maybe we need to associate the title with the state better so we can push/pop titles instead of actively setting them in the component. Opinions?